templates/apollo-graphql-react/files/.platform/routes.yaml

```
@@ -0,0 +1,24 @@
+# The routes of the project.
+#
+# Each route describes how an incoming URL is going
+# to be processed by Platform.sh.
+
+"https://www.{default}/":
+    type: upstream
+    upstream: "front:http"
+
+"https://{default}/":
+    type: redirect
+    to: "https://www.{default}/"
+
+"https://api.{default}/":
+    type: upstream
+    id: graphqlEntrypoint
```

I think IDs are supposed to be lowercase, no?  We generally use kebab case for these.

```suggestion
    id: graphql-entrypoint
```

Though I admit I detest the word "entrypoint". It's such a meaningless buzzword.

---

The documentation doesn't specify a specific ID format, although the provided example is in kebab case, indeed. IDs being strings, it shouldn't change anything no matter what convention is used?
In JavaScript, camelCase is preferred. This mainly comes from the fact that identifiers can't contain dashes. Regarding object keys with a special characters, they can be accessed using the bracket notation: `obj["key-with-special-chars"]`, which is less convenient than the dot notation: `obj.keyWithoutSpecialChars`.
So to keep things consistent, I like having everything the same rather than wondering what typographical convention a value (that I have control over) uses.

If there are strong feelings about having ID values in camelCase, I'll change to kebab-case and adapt the code accordingly.